sigfox-config.yml

```diff
@@ -0,0 +1,7 @@
+# Configuration fields and values
+sigfox_api:
+  login: <login_id>
+  pwd: <password>
```

Comment:
We should not encourage users to edit files that are stored in git. The better way to handle this is to call this file `sigfox-config-template.yml`, and have an install step that copies *and populates* `sigfox-config.yml` from `sigfox-config-template.yml`. Then make sure `.gitignore` explicitly excludes `sigfox-config.yml`. (To make this even better, do this like the `key.d` directory in more recent signing bootloader builds; make sure the directory is user-readable only, and make sure the password file is user-readable only. Even more robust is to use [`pynacl`](https://pypi.org/project/PyNaCl/) (docs [here](https://pynacl.readthedocs.io/en/latest/]) and use `nacl.secret.box.encrypt()`/`nacl.secret.box.decrypt()` plus a prompt to the user to give us their password when decrypting the key prior to transmitting it to Sigfox.  

At the very least, implement the template/`.gitignore` portion.

Reply:
I am working on these changes and will update you soon.